modules/deploy/partials/requirements.adoc

```diff
@@ -52,25 +52,17 @@ ifndef::env-kubernetes[]
 endif::[]
 
 [[node-updates]]
-== Node maintenance and operating system upgrades
+== Prevent automatic node upgrades
 
 Ensure that node and operating system (OS) upgrades are manually managed when running Redpanda in production. Manual control avoids unplanned reboots or replacements that disrupt Redpanda brokers, causing service downtime, data loss, or quorum instability.
 
-=== Limitations of automatic updates
+Common issues with automatic node upgrades include:
 
-Redpanda is stateful. Redpanda brokers manage partition data and leadership, making them sensitive to disruptions. Proper handling during maintenance is required to:
-
-- Avoid data loss, especially for nodes with ephemeral or local storage.
-- Ensure smooth leadership transitions by decommissioning brokers before removing a node.
-- Minimize service downtime by upgrading nodes one at a time during planned maintenance windows.
-
-However, automatic update mechanisms provided by cloud platforms may not meet Redpanda's stateful requirements. Common issues include:
-
-- Hard timeouts for graceful shutdowns that may not allow Redpanda brokers enough time to complete decommissioning or leadership transitions.
+- Hard timeouts for graceful shutdowns that do not allow Redpanda brokers enough time to complete decommissioning or leadership transitions.
 - Replacements or reboots without ensuring data has been safely migrated or replicated, risking data loss.
 - Parallel upgrades across multiple nodes, which can disrupt quorum or reduce cluster availability.
 
-*Recommendations*:
+*Requirements*:
```

Comment:
Not important, but I think the 3 links below look a little odd with just "Azure documentation." Suggest: "See the Azure documentation." for each link